code/modules/mob/living/carbon/human/species_types/IPC.dm

```diff
@@ -216,12 +216,17 @@
 	H.notify_ghost_cloning("You have been repaired!")
 	H.grab_ghost()
 	H.dna.features["ipc_screen"] = "BSOD"
+	INVOKE_ASYNC(src, .proc/declare_revival, H)
+	
 	H.update_body()
-	playsound(H, 'sound/voice/dialup.ogg', 25)
+	return
+
+/datum/species/ipc/proc/declare_revival(mob/living/carbon/human/H)
```

Comment:
this change is good but seems out of scope

Reply:
As it turns out it's not, gave me a bunch of errors when I reverted it.